metalearners/metalearner.py

```diff
@@ -825,8 +864,40 @@ def evaluate(
         w: Vector,
         is_oos: bool,
         oos_method: OosMethod = OVERALL,
-    ) -> dict[str, float | int]:
-        """Evaluate all models contained in a MetaLearner."""
+        scoring: Mapping[str, list[str | Callable]] | None = None,
+    ) -> dict[str, float]:
+        r"""Evaluate the MetaLearner.
+
+        The keys in ``scoring`` which are not a name of a model contained in the MetaLearner
+        will be ignored, for information about this names check
+        :meth:`~metalearners.metalearner.MetaLearner.nuisance_model_specifications` and
+        :meth:`~metalearners.metalearner.MetaLearner.treatment_model_specifications`.
+        The values must be a list of:
+
+        * ``string`` representing a ``sklearn`` scoring method. Check
+          `here <https://scikit-learn.org/stable/modules/model_evaluation.html#common-cases-predefined-values>`__
```

Comment:
```suggestion
          `here <https://scikit-learn.org/stable/modules/model_evaluation.html#common-cases-predefined-values>`_
```

Reply:
This is to avoid a warning when building the docs that the reference already exists (as there's another link which is reference with `here` also), see [here](https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use).